site/reference/pkg/init/README.md

```diff
@@ -46,39 +58,14 @@ cd $TEST_HOME
 
 <!-- @pkgInit @verifyStaleExamples-->
 ```shell
-# writes Kptfile package meta if not found
+# Creates a new Kptfile with metadata in the my-pkg directory.
 mkdir my-pkg
-kpt pkg init my-pkg --tag kpt.dev/app=cockroachdb \
+kpt pkg init my-pkg --keyWords kpt.dev/app=cockroachdb \
```

Comment:
Let's pick something human redable for keyWords:
cockroachdb, nosql, db, etc.

Reply:
yeah, updated the example here.